kernel/power/gemm_common.c

```diff
@@ -28,7 +28,12 @@
 #define USE_VECTOR_PAIRS
 #endif
 
+#ifdef _AIX
+#include<stdbool.h>
+typedef __vector unsigned short vec_bf16;
```

Comment:
I don't think we need a separate typedef for AIX.  I believe the original should work.